executor/importer/table_import.go

```diff
@@ -147,6 +147,29 @@ func GetCachedKVStoreFrom(pdAddr string, tls *common.TLS) (tidbkv.Storage, error
 	return kvStore, nil
 }
 
+// GetRegionSplitSizeKeys gets the region split size and keys from PD.
+func GetRegionSplitSizeKeys(ctx context.Context) (regionSplitSize int64, regionSplitKeys int64, err error) {
+	tidbCfg := tidb.GetGlobalConfig()
```

Comment:
maybe in future we can get the singleton PD client of this TiDB instance, and use the latest PD address, in case of PD node is changed.